tools/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -805,6 +807,161 @@ public void generateSummaryHtmlErrorTables(Appendable output, EnumSet<Choice> ch
 
     }
 
+    private class WriteContext {
+
+        private List<String> names = new ArrayList<>();
```

Comment:
I'm guessing these are _localeNames_ and _localeIds_

I know our code has lots of bad names, but for new code we should be clearer.